src/main/java/com/stripe/model/Source.java

```diff
@@ -18,13 +18,13 @@
 @EqualsAndHashCode(callSuper = false)
 public class Source extends ApiResource implements PaymentSource, MetadataStore<Source> {
   @SerializedName("ach_credit_transfer")
-  SourceTypeAchCreditTransfer achCreditTransfer;
+  AchCreditTransfer achCreditTransfer;
 
   @SerializedName("ach_debit")
-  SourceTypeAchDebit achDebit;
+  AchDebit achDebit;
```

Comment:
technically, AchDebit is not public on Sources

Reply:
@remi-stripe 
Does it makes sense that `/v1/topups/:topup/cancel` actually returns `ach_debit` type? I see that in the backend test.

Reply:
sadly yes.

Reply:
Feel free to just leave as is, it's okay